build/CODE_OF_CONDUCT.md

```diff
@@ -1,3 +0,0 @@
-## Community Code of Conduct
```

Comment:
Is this intentional?

Reply:
Yes, the build folder was previously not a `submodule`. This PR deletes old folder and adds it as a submodule (see the PR description)